crates/api_models/src/analytics/refunds.rs

```diff
@@ -24,8 +24,8 @@ use crate::enums::{Currency, RefundStatus};
 #[serde(rename_all = "snake_case")]
 #[strum(serialize_all = "snake_case")]
 pub enum RefundType {
```

Comment:
@ShivanshMathurJuspay Why do we have this enum in the `analytics` crate as well? And does it need to derive `Default`?